etc/docker/dev/docker-compose-storage-monit.yml

```diff
@@ -0,0 +1,128 @@
+version: "2"
+services:
+  rucio:
+    image: rucio/rucio-dev
+    hostname: rucio
+    ports:
+      - "443:443"
+    links:
+      - ruciodb:ruciodb
+      - graphite:graphite
+      - fts:fts
+      - ftsdb:ftsdb
+      - xrd1:xrd1
+      - xrd2:xrd2
+      - xrd3:xrd3
+      - activemq:activemq
+    volumes:
+      - ../../../tools:/opt/rucio/tools
+      - ../../../bin:/opt/rucio/bin
+      - ../../../lib:/opt/rucio/lib
+    environment:
+      - X509_USER_CERT=/opt/rucio/etc/usercert.pem
+      - X509_USER_KEY=/opt/rucio/etc/userkey.pem
+  ruciodb:
+    image: postgres:11
+    environment:
+      - POSTGRES_USER=rucio
+      - POSTGRES_DB=rucio
+      - POSTGRES_PASSWORD=secret
```

Comment:
Putting DB data on volume is better in case of restart / kill containers. It can avoid to recreate it. e.g. 
```
volumes:
      - pg_data:/var/lib/postgresql/data
volumes:
  pg_data:
    external: true
  mysql_data:
    external: true
```

Reply:
we had a couple of cases where changes to the database schema were screwing with the dev environment if we didn't start with a clean slate. most likely the best approach would be a separate docker-compose that explicitly puts this on a volume, so people can use that when they want.